Source/APIClient/Request.swift

```diff
@@ -18,13 +18,15 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var version: String { get }
+    static var serverAddress: String? { get }
```

Comment:
Is this the right name for this? Also, should it be `static`, i.e. are you likely to only have one of these per Request class?